proms-resources/frontend/src/main/frontend/src/proms/ToUDialog.jsx

```diff
@@ -122,7 +122,7 @@ function ToUDialog(props) {
     return null;
   }
 
-  if (tou && touAcceptedVersion && tou.version == touAcceptedVersion) {
+  if (tou && (touAcceptedVersion && tou.version == touAcceptedVersion || !tou.enabled) ) {
```

Comment:
It reads more logically if we swap:
```suggestion
  if (tou && (!tou.enabled || touAcceptedVersion && tou.version == touAcceptedVersion) ) {
```